avocado/plugins/run.py

```diff
@@ -152,8 +152,11 @@ def configure(self, parser):
                              help="Inject [path:]key:node values into the "
                              "final multiplex tree.")
 
-    def _activate(self, args):
-        # Extend default multiplex tree of --mux_inject values
+    @staticmethod
+    def _process_mux_inject(args):
```

Comment:
Hello @clebergnu, it was copied over from other plugin, where different kind of `activate` functions were bundled. There are actually multiple differences in `multiplex` vs. `run`, I addressed some of them https://github.com/avocado-framework/avocado/pull/1437 I haven't changed the  `activate`, though.